pkgs/applications/networking/bee/ensure-clef-account

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+
+# NOTE This file is called by the systemd service in its preStart
+# hook, but it's not Nix specific in any way. Ideally, the same file
+# should be called from the postinst scripts of the other packages,
+# but... the world is not ideal.
+
+# What follows was extracted from, and should be in sync with
+# https://github.com/ethersphere/bee-clef/tree/master/packaging
```

Comment:
Either install from the source distribution or use `fetchurl`, no need to vendor this.

Reply:
this file is not available in upstream. there was a short debate where to put it, but i decided that it's better here, because this way there's no need for synchronizing with the upstream repo (i don't have the commit bit there, and my PR to add this file has met some resistance).